libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SqliteLibraryLoaderTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015 Salesforce.com.
```

Comment:
Change copyright to `Mobile SDK` copyright. You should be able to get it from one of the other files.